doc/lua_api.md

```diff
@@ -4119,6 +4137,14 @@ The file should be a text file, with the following format:
   There must be no extraneous whitespace around the `=` or at the beginning or
   the end of the line.
 
+Gettext translation file format
+-------------------------------
+
+Gettext files can also be used as translations. A translation file has the suffix
+`.[lang].po` or `.[lang].mo`, depending on whether it is compiled or not, and must
+also be placed in the `locale` subdirectory of the mod. The value of `textdomain`
+is `msgctx` in the gettext files.
```

Comment:
Wouldn't it make sense to also allow placing `.po` files in the `po/` subdirectory of the mod?

Reply:
```suggestion
is `msgctxt` in the gettext files.
```

Reply:
Please add an example:

```po
msgctxt "textdomain"

msgid "Hello world!"
msgstr "Bonjour le monde!"
```